tests/test_utils.py

```diff
@@ -0,0 +1,167 @@
+import time
+from unittest.mock import patch
+
+import pytest
+
+from accept import AcceptUtils
+
+
+@pytest.fixture
+def accept_utils_instance():
+    return AcceptUtils()
+
+
+class TestGenerateMerchantOrderId:
+    def test_generate_merchant_order_id(self, accept_utils_instance):
```

Comment:
After adding only one Test class for each class the test cases naming convention will be like this:
```
def test_<function_name>_<short description of what we are testing in the function>:
    """test <function name>: long description of what we are testing"""
```